obelisk_ws/src/python/obelisk_leap_py/obelisk_leap_py/obelisk_leap_hand.py

```diff
@@ -0,0 +1,16 @@
+from typing import List, Optional
+
+from rclpy.executors import MultiThreadedExecutor
+
+from obelisk_py.zoo.robot.hardware.leap_hand.leap_node import ObeliskLeapHand
+from obelisk_py.core.utils.ros import spin_obelisk
+
+
+def main(args: Optional[List] = None) -> None:
+    """Main entrypoint."""
+    spin_obelisk(args, ObeliskLeapHand, MultiThreadedExecutor)
```

Comment:
does this executor need to be multithreaded? i don't actually think it does (please discuss multi vs single threaded executors with zach to get an overview)

Reply:
Changed to single threaded.